Source/Core/Core/PowerPC/JitCommon/JitBase.cpp

```diff
@@ -42,7 +42,6 @@ bool JitBase::MergeAllowedNextInstructions(int count)
 void JitBase::UpdateMemoryOptions()
 {
   bool any_watchpoints = PowerPC::memchecks.HasAny();
-  jo.fastmem = SConfig::GetInstance().bFastmem && !any_watchpoints;
+  jo.fastmem = SConfig::GetInstance().bFastmem;
   jo.memcheck = SConfig::GetInstance().bMMU || any_watchpoints;
-  jo.alwaysUseMemFuncs = any_watchpoints;
```

Comment:
Oh, I think I was wrong here. We still need
`jo.fastmem = SConfig::GetInstance().bFastmem && (MSR.DR || !any_watchpoints);`

We don't hide those pages when virtual memory is disabled. This change likely won't change the performance, but still supports memcheck if the MMU is disabled